layouts/partials/comments/provider/remark42.html

```diff
@@ -0,0 +1,60 @@
+<div id="remark42">
+
+</div>
+<script>
+    var remark_config = {
+      host: "{{ .Site.Params.comments.remark42.host }}", // hostname of remark server, same as REMARK_URL in backend config, e.g. "https://demo.remark42.com"
+      site_id: '{{ .Site.Params.comments.remark42.site }}',
```

Comment:
Use `.Site.Params.comments.remark42.site_id` instead of `.Site.Params.comments.remark42.site`.